mission3/src/comment/commands/create-comment/create-comment.service.js

```diff
@@ -0,0 +1,11 @@
+export const createCommentService = async (commentData) => {
+  return await prisma.comment.create({
```

Comment:
프리즈마에 정의하신 모델 명과 다른 모델 명이라 에러가 발생할 것 같아요.
```suggestion
  return await prisma.userComment.create({
```